### DISCUSSION TITLE
VFR Plan Altitude between 2 WP

### DISCUSSION THREAD

**Discussion by didierma34** [albar965/littlenavmap#835] [Q&A]
Hello,

Is it possible to define a different altitude between 2 WP?
In VFR, depending on the areas to be crossed, we must plan for different altitudes.
If this is possible, what should be done?

Example :
- Aerodrome departure 0ft
- Aerodorm departure towards WP1 at 2000 ft
- WP1 to WP2 at 3000 ft
- WP2 to WP3 at 1500 ft
etc ...

Thank's

**Comment by albar965** [comment]
Sorry, LNM does not support this now. There is only one cruise altitude. 

Unfortunately this is not doable with reasonable effort.

**Reply by didierma34** [reply]
ok...

Thank's and good Job !!
